stale_repos.py

```diff
@@ -218,12 +225,13 @@ def output_to_json(inactive_repos, file=None):
     #   }
     # ]
     inactive_repos_json = []
-    for repo_url, days_inactive, last_push_date in inactive_repos:
+    for repo_url, days_inactive, last_push_date, private in inactive_repos:
         inactive_repos_json.append(
             {
                 "url": repo_url,
                 "daysInactive": days_inactive,
                 "lastPushDate": last_push_date,
+                "visibility": private,
```

Comment:
```suggestion
                "visibility": visibility,
```

Reply:
Added fix and set you as co-author.  Thank you for feedback